backend/integrations/discord/bot.py

```
@@ -67,9 +69,50 @@ async def on_message(self, message):
+                if proactive_type == "greeting":
+                    await message.channel.send(
+                        f"Hi {message.author.mention}! 👋\n"
+                        "Welcome to the community!\n"
+                        "If you're new, I can guide you on how to start contributing 🚀"
+                    )
+                    return
+
+                if proactive_type == "onboarding":
+                    await message.channel.send(
+                        f"Awesome {message.author.mention}! 🎉\n"
+                        "Here’s how you can start:\n"
```

Ruff (0.15.1)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)